test/TestCases/Case04/case.json

```diff
@@ -2,11 +2,11 @@
     "name": "Get Database Schema",
     "description":"",
     "argv":["--quiet",
-            "--input-graphdb-schema-neptune-endpoint", "airport00.cluster-ccbtmrye8gfu.us-east-1.neptune.amazonaws.com:8183",            
+            "--input-graphdb-schema-neptune-endpoint", "g-hgb6zpdbh5.us-west-2.neptune-graph.amazonaws.com:8183",            
             "--output-folder-path", "./test/TestCases/Case04/output",
             "--output-no-lambda-zip"],
-    "host": "",
-    "port": "",
+    "host": "g-hgb6zpdbh5.us-west-2.neptune-graph.amazonaws.com",
```

Comment:
Please remove all endpoints. GitHub history should also be cleaned up to remove references to endpoints.